frontend/dashboard/components/ResourcesRepoList/ResourcesRepoList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { RESOURCEADM_BASENAME } from 'app-shared/constants';
+import { useSelectedContext } from 'dashboard/hooks/useSelectedContext';
+import { ResourceTable } from 'resourceadm/components/ResourceTable';
+import { useGetResourceListQuery } from 'resourceadm/hooks/queries/useGetResourceListQuery';
+import { getResourceDashboardURL, getResourcePageURL } from 'resourceadm/utils/urlUtils';
+import { getReposLabel } from 'dashboard/utils/repoUtils';
+import { Organization } from 'app-shared/types/Organization';
+import { useTranslation } from 'react-i18next';
+import { SelectedContextType } from 'app-shared/navigation/main-header/Header';
+import { AltinnSpinner } from 'app-shared/components';
+
+type ResourcesRepoListProps = {
+  organizations: Organization[];
+};
+
+export const ResourcesRepoList = ({ organizations }: ResourcesRepoListProps): React.ReactNode => {
+  const { t } = useTranslation();
+  const selectedContext = useSelectedContext();
+  const repo = `${selectedContext}-resources`;
+
+  const isOrganisation =
```

Comment:
Looks like we write organi**z**ation with z other places within the codebase. The prop is also written with "z". :)
```suggestion
  const isOrganization=
```